src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+public class Event extends Task {
+    protected String at;
```

Comment:
Just like the Deadline class, perhaps you can change the variable name from "at" to "eventTime"?